evap/evaluation/templates/contribution_formset.html

```diff
@@ -52,7 +52,7 @@
                 </td>
                 {% if editable %}
                     <td>
-                        {% if staff or not form.responsibility.value == "RESPONSIBLE" %}
+                        {% if staff or responsible %}
```

Comment:
what was the intention behind this? because right now it checks whether the `responsible` variable contains something, which it always does.